cmd/zetacored/parse_genesis.go

```diff
@@ -0,0 +1,189 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"cosmossdk.io/math"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	vestingtypes "github.com/cosmos/cosmos-sdk/x/auth/vesting/types"
+	"github.com/cosmos/cosmos-sdk/x/authz"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	crisistypes "github.com/cosmos/cosmos-sdk/x/crisis/types"
+	distributiontypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
+	evidencetypes "github.com/cosmos/cosmos-sdk/x/evidence/types"
+	"github.com/cosmos/cosmos-sdk/x/genutil"
+	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	"github.com/cosmos/cosmos-sdk/x/group"
+	paramstypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	slashingtypes "github.com/cosmos/cosmos-sdk/x/slashing/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+	evmtypes "github.com/evmos/ethermint/x/evm/types"
+	feemarkettypes "github.com/evmos/ethermint/x/feemarket/types"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/tendermint/types"
+	"github.com/zeta-chain/zetacore/app"
+	crosschaintypes "github.com/zeta-chain/zetacore/x/crosschain/types"
+	emissionsModuleTypes "github.com/zeta-chain/zetacore/x/emissions/types"
+	fungibleModuleTypes "github.com/zeta-chain/zetacore/x/fungible/types"
+	observertypes "github.com/zeta-chain/zetacore/x/observer/types"
+)
+
+const MaxItemsForList = 10
+
+var Copy = map[string]bool{
+	slashingtypes.ModuleName:        true,
+	govtypes.ModuleName:             true,
+	crisistypes.ModuleName:          true,
+	feemarkettypes.ModuleName:       true,
+	paramstypes.ModuleName:          true,
+	upgradetypes.ModuleName:         true,
+	evidencetypes.ModuleName:        true,
+	vestingtypes.ModuleName:         true,
+	fungibleModuleTypes.ModuleName:  true,
+	emissionsModuleTypes.ModuleName: true,
+	authz.ModuleName:                true,
+}
+var Skip = map[string]bool{
+	evmtypes.ModuleName:          true,
+	stakingtypes.ModuleName:      true,
+	genutiltypes.ModuleName:      true,
+	authtypes.ModuleName:         true,
+	banktypes.ModuleName:         true,
+	distributiontypes.ModuleName: true,
+	group.ModuleName:             true,
+}
+
+var Modify = map[string]bool{
+	crosschaintypes.ModuleName: true,
+	observertypes.ModuleName:   true,
```

Comment:
Can we add comments what does it represent

Reply:
b8d51418bf8bd06e46373ace546bd62af7508cf2